### PR TITLE
to avoid passing unsupported parameter

### PR DESCRIPTION
**Pull request by etrex**
Remove options.certificate of Telegram#setWebhook before sending the request with testing code.

### CODE REVIEW

File: packages/messaging-api-telegram/src/TelegramClient.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import omit from 'lodash.omit';
 import snakeCaseKeys from 'snakecase-keys';
 import urlJoin from 'url-join';
 import { onRequest } from 'messaging-api-common';
+import { pick } from 'lodash';
```
Comment (jigsawye) [inline packages/messaging-api-telegram/src/TelegramClient.ts]: Should we change this to `import pick from 'lodash/pick';`?

cc @chentsulin

Comment (chentsulin) [reply packages/messaging-api-telegram/src/TelegramClient.ts]: I would vote for 

```js
import pick from 'lodash/pick';
```

Comment (etrex) [reply packages/messaging-api-telegram/src/TelegramClient.ts]: Fixed.

File: packages/messaging-api-telegram/src/TelegramClient.ts [hunk 2]
```diff
@@ -149,9 +150,14 @@ export default class TelegramClient {
    * @returns True on success.
    */
   setWebhook(url: string, options: SetWebhookOption = {}): Promise<boolean> {
+    const snakecaseOptions = snakeCaseKeys(options);
```
Comment (chentsulin) [inline packages/messaging-api-telegram/src/TelegramClient.ts]: Should we do this transform here? Or just apply a deep snake case transform in `this._request`?

Comment (etrex) [reply packages/messaging-api-telegram/src/TelegramClient.ts]: cool

Comment (etrex) [reply packages/messaging-api-telegram/src/TelegramClient.ts]: updated

**Pull request merged**